Source/DiscoveryV1/Discovery.swift

```diff
@@ -2710,6 +2716,79 @@ public class Discovery {
         request.responseObject(completionHandler: completionHandler)
     }
 
+    /**
+     Get Autocomplete Suggestions.
+
+     Returns completion query suggestions for the specified prefix.  /n/n **Important:** this method is only valid when
+     using the Cloud Pak version of Discovery.
+
+     - parameter environmentID: The ID of the environment.
+     - parameter collectionID: The ID of the collection.
+     - parameter `prefix`: The prefix to use for autocompletion. For example, the prefix `Ho` could autocomplete to
+       `Hot`, `Housing`, or `How do I upgrade`. Possible completions are.
+     - parameter field: The field in the result documents that autocompletion suggestions are identified from.
+     - parameter count: The number of autocompletion suggestions to return.
+     - parameter headers: A dictionary of request headers to be sent with this request.
+     - parameter completionHandler: A function executed when the request completes with a successful result or error
+     */
+    public func getAutocompletion(
+        environmentID: String,
+        collectionID: String,
+        `prefix`: String,
```

Comment:
Very interesting to see this reserved word escaping on a parameter!

Reply:
I was gonna mention that too but then I noticed `return` in the `Query` method